src/components/forms/MultiSelectInput.jsx

```diff
@@ -7,7 +7,11 @@ const ReferenceSelectArrayInputItem = ({ referenceValues }) => {
     <>
       {
         referenceValues && (
-        <ReferenceArrayInput reference={reference} source={source}>
+        <ReferenceArrayInput 
+          reference={reference}
+          source={source}
+          perPage={50}
```

Comment:
As we discussed, up to 50 items for a ListBox is too much to have on the screen, and if we eventually have more items we will have the same issue. We will go with these changes to fix the bug for now, but we need to improve this component to allow scrolling through the items or use an autocomplete function.

Reply:
Removed the sort by name, ready for hotfix